app/keepers/keepers.go

```diff
@@ -135,6 +137,7 @@ type AppKeepers struct {
 	RouterKeeper             *routerkeeper.Keeper
 	RatelimitKeeper          ratelimitmodulekeeper.Keeper
 	AllianceKeeper           alliancemodulekeeper.Keeper
+	XCvmKeeper               xcvmkeeper.Keeper
```

Comment:
nit: typical go style is to capitalize abbreviations
```suggestion
	XCVMKeeper               xcvmkeeper.Keeper
```